### ISSUE TITLE
data source

### ISSUE THREAD

**Issue by TankardYegga** [julianschoep/SGRLayer#2]
May I ask how can I get the class hierarchy graph of the symbolic nodes?

**Comment by julianschoep** [comment]
@LevinLittleDream Yes, my bad for not adding it. I added everything you need to construct the hierarchy, incl. node names/mappings in the latest commit d2c8bae
